media/materials/programs/point_light_shadow_demo_fp.glsl

```diff
@@ -0,0 +1,64 @@
+#version 130
+
+uniform sampler2DShadow shadowMap0;
+uniform sampler2DShadow shadowMap1;
+uniform sampler2DShadow shadowMap2;
+uniform sampler2DShadow shadowMap3;
+uniform sampler2DShadow shadowMap4;
+uniform sampler2DShadow shadowMap5;
+
+uniform vec4 lightPos0;
```

Comment:
I think we can remove this var? looks like it's not being used

Reply:
https://github.com/osrf/gazebo/pull/3051/commits/240f51c28ab67c45cec8a01781d47420603167e2